api/client/proto/authservice.proto

```diff
@@ -156,15 +156,18 @@ message Certs {
     repeated bytes SSHCACerts = 4 [ (gogoproto.jsontag) = "ssh_ca_certs,omitempty" ];
 }
 
-// GenerateServerKeysRequest specifies certificate-generation parameters
+// HostCertsRequest specifies certificate-generation parameters
 // for a server.
-message GenerateServerKeysRequest {
+message HostCertsRequest {
```

Comment:
nit: GenerateHostCertsRequest, so it matches the RPC name?

```suggestion
message GenerateHostCertsRequest {
```

Reply:
We already have `rpc GenerateUserCerts` that takes a `UserCertsRequest` so I was trying to be consistent with existing RPCs.

Reply:
Consistency is not our strong suite (other parts of the gRPC API go for RPC Foo / FooRequest / FooResponse) - I stand by my suggestion, as I think it's the better practice, but up to you.

Reply:
I agree with this change.